jf_agent/git/__init__.py

```diff
@@ -427,6 +429,12 @@ def get_repos_from_git(git_connection, config: GitConfig):
         projects = gl_adapter.get_projects()
         repos = gl_adapter.get_repos(projects)
     else:
+        agent_logging.log_and_print_error_or_warning(
+            logger,
+            logging.ERROR,
+            msg_args=[config.git_provider],
+            error_code=3071,
+            exc_info=True,
+        )
```

Comment:
This is something we didn't used to write a log message for; it'd be OK to continue to not write a log message for it, as this is essentially a fatal error that will kill the run, and since I believe that'd be more consistent with how we handle other fatal-type errors.